extensions-core/parquet-extensions/src/main/java/org/apache/druid/data/input/parquet/simple/ParquetGroupConverter.java

```diff
@@ -492,10 +510,12 @@ private static BigDecimal convertBinaryToDecimal(Binary value, int precision, in
   }
 
   private final boolean binaryAsString;
+  private final boolean convertCorruptDates;
 
-  public ParquetGroupConverter(boolean binaryAsString)
+  public ParquetGroupConverter(boolean binaryAsString, boolean convertCorruptDates)
   {
     this.binaryAsString = binaryAsString;
```

Comment:
is `binaryAsString` still used somewhere?

Reply:
yes; it should be used around [here](https://github.com/apache/druid/blob/c3bab1685db50d03809e33d249c47aad594ef043/extensions-core/parquet-extensions/src/main/java/org/apache/druid/data/input/parquet/simple/ParquetGroupConverter.java#L436)